File: Problem_set_1/Introductory_problems/p02_Missing_Numbers/missingNumbers.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
    long long n;
    cin >> n;

    vector <long long> number(n - 1);

    long long actual_sum = 0;

    for(int i = 0; i < n - 1; i++){
        cin >> number[i];
        actual_sum += number[i];
    }

    long long expected_sum = n * (n + 1) / 2;

    cout << expected_sum - actual_sum;
    return 0;
}

File: Problem_set_1/Introductory_problems/p02_Missing_Numbers/missingNumbers.ts
const prompt = require("prompt-sync")();

const n: number = Number(prompt());

const numbers: number[] = prompt()
    .split(" ")
    .map(Number);

let actualSum = 0;

for (const num of numbers) {
    actualSum += num;
}

const expectedSum = n * (n + 1) / 2;

console.log(expectedSum - actualSum);

export {};